local-cli/library/library.js

```diff
@@ -59,7 +59,8 @@ function _library(argv, config, resolve, reject) {
       return;
     }
 
-    const dest = f.replace(/Sample/g, args.name).replace(/^_/, '.');
+    const dest = f.replace(path.join(source, path.sep),'').replace(/Sample/g, args.name).replace(/^_/, '.');
```

Comment:
Would you find using `path.relative` here more readable that trimming using `replace`?
https://nodejs.org/api/path.html#path_path_relative_from_to

Similar to https://github.com/facebook/react-native/blob/3aa25f79172159e3079568d0131623054916fb59/local-cli/generator/copyProjectTemplateAndReplace.js#L41